README.md

```diff
@@ -1,3 +1,49 @@
 # azure-file-csi-driver-operator
 
 An operator to deploy the [Azure File CSI Driver](https://github.com/openshift/azure-file-csi-driver) in OKD.
+
+This operator is installed by the [cluster-storage-operator](https://github.com/openshift/cluster-storage-operator).
+
+
+
+origin-cluster-cloud-controller-manager-operator
```

Comment:
This looks very strange

Reply:
/lgtm cancel

Reply:
Thanks, that got in unintentionally, removing.

Reply:
@jsafrane PTAL again and add lgt if you're ok with the change.